Classes/Services/MetsExporter.php

```diff
@@ -793,6 +795,16 @@ public function getSlubData()
     }
 
     /**
+     * replaces single quote with double quote
+     * @param $string
+     * @return string
+     */
+    function replaceSingleQuote($string)
```

Comment:
When making the method static it would be replaced by `self::`, which is one character less than `$this->`. One could make the code even smaller by inlining the `str_replace` function call, but readability would suffer and the compiler is probably doing this anyway.

Not sure about performance, but it seems like an attempt to speed up a car by throwing out the ashtray.

BTW: The `$this` pseudo-variable is not a function argument. And it is very explicit since it is written in the code!